net-misc/modemmanager/modemmanager-1.14.8.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
There is also an EAPI 7 update for this in #18813.

Reply:
Thanks! I've incorporated the changes in https://github.com/gentoo/gentoo/pull/18813/commits/b941b20490fe11a27e41b6d5dfd504bd6767cf6b